src/packet.ml

```diff
@@ -395,6 +394,25 @@ module Tls_crypt = struct
         ack_message_ids;
         remote_session;
       }
+    in
+    (res, 1 + (arr_len * packet_id_len) + rs_len)
```

Comment:
the `1` is the byte of the `arr_len` field?

Reply:
Yes

Reply:
that may deserve a comment or not, your take :)